maker-core/delegates.md

```diff
@@ -1,5 +1,39 @@
-# TBC
+# Constitutional Delegates (CDs)
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Overview
+Delegates are ecosystem actors who operate through Protocol Delegation System smart contracts. MKR holders can delegate their voting power to them without the need to hand over the custody of their MKR.
+
+Constitutional Delegates (CDs) are one type of [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). They receive various benefits and are subject to specific requirements. They must put in place the governance strategies defined by the CVCs.
+
+CDs must not engage with other ecosystem actors or in operational activities. They are not allowed to compensate MKR holders that delegate to them. They must aim to provide governance information and research material to MKR holders.
```

Comment:
They must aim to provide governance information and research material to MKR holders - is this not done by CVCs? Or is it both?

Reply:
I will change it for "CVCs."

From MIP101 latest amendments:

##### 6.5: Constitutional Delegate Alignment Risk Mitigation

CDs are generally not allowed to engage directly with Ecosystem Actors and other counterparties to the DAO, or in other ways engage in operational activities. They must remain focused on publicly providing governance information and research material to CVCs, and can interact with ecosystem participants for this sole purpose